posthog/hogql/hogql.py

```diff
@@ -311,43 +322,67 @@ def translate_ast(node: ast.AST, stack: List[ast.AST], context: HogQLContext) ->
     return response
 
 
-def property_access_to_clickhouse(chain: List[str]):
+def parse_field_access(chain: List[str], context: HogQLContext) -> HogQLFieldAccess:
     # Circular import otherwise
     from posthog.models.property.util import get_property_string_expr
 
     """Given a list like ['properties', '$browser'] or ['uuid'], translate to the correct ClickHouse expr."""
     if len(chain) == 2:
         if chain[0] == "properties":
+            if context.values is not None:
+                key = f"val_{len(context.values)}"
```

Comment:
let's maybe get even more unique here? would a) help debug b) ensure we don't mess up in the future.

Why not `hogql_val_` as a prefix?

Reply:
Yeah, let's be explicit with the name, `hogql_val_` works for me.